pkg/claimsstore/pgstore.go

```diff
@@ -0,0 +1,77 @@
+package claimsstore
+
+import (
+	"github.com/iden3/go-iden3-core/db"
+	"github.com/jinzhu/gorm"
+)
+
+// PGStore is an implementation of the iden3 storage interface that uses postgres as its backend
+type PGStore struct {
```

Comment:
Perhaps not right now, but seems like this would be a good opportunity to give back to them and submit a PR into `iden3-core` as another db implementation.